Doc/tools/extensions/pyspecific.py

```diff
@@ -253,7 +253,7 @@ def run(self):
             text = 'The NEWS file is not available.'
             node = nodes.strong(text, text)
             return [node]
-        content = issue_re.sub(r'`\1ssue #\2 <https://bugs.python.org/\2>`__',
+        content = issue_re.sub(r'`bpo-\1 <https://bugs.python.org/\1>`__',
```

Comment:
I'd suggest making it `https://bugs.python.org/issue\1` to avoid the redirect through https://www.python.org/sf/